Backend/app.py

```
@@ -174,15 +188,23 @@ def cleanup_cache():
     return appointment_db
 
+
+@app.route("/health", methods=["GET"])
+def health():
+    return {"status": "ok", "agent_initialized": AGENT_INITIALIZED}
+
+
+# Run server
 if __name__ == '__main__':
-   app.run(host='0.0.0.0', port=5000, debug=True)
+    app.run(host='0.0.0.0', port=5000, debug=True)
```

Ruff (0.14.10)

Possible binding to all interfaces

(S104)